termios_linux.go

```diff
@@ -8,32 +8,3 @@ const (
 	getTermios = unix.TCGETS
```

Comment:
Maybe have `termios_unix.go` (or `termios_non-bsd.go`, IDK) with `//+build !darwin,!freebsd,!openbsd,!netbsd,!windows` instead of having two files with the same contents?

Reply:
Agree. Decided to name the file `termios_nonbsd.go` because in many cases `_unix.go` implies all the unices (including the BSDs).